src/validate/validate_global_state_expression.test.ts

```diff
@@ -0,0 +1,69 @@
+import {createExpression} from '..';
+import {findGlobalStateExpressionKeys} from './validate_global_state_expression';
+
+test('findGlobalStateExpressionKeys extracts all global state keys', () => {
```

Comment:
Use `describe` for the test suite please.

Reply:
Done